plugins/inputs/kernel/kernel.go

```diff
@@ -145,12 +156,18 @@ func (k *Kernel) Gather(acc telegraf.Accumulator) error {
 	}
 	acc.AddCounter("kernel", fields, map[string]string{})
 
+	if k.optCollect["psi"] {
+		if err := k.gatherPressure(acc); err != nil {
+			return err
```

Comment:
Let's make the error specific here to easily find the spot on debugging...
```suggestion
			return fmt.Errorf("gathering pressure failed: %w", err)
```

Reply:
I've reverted this suggestion, since there's only one routine in `k.gatherPressure` that [returns an already-formatted error](https://github.com/influxdata/telegraf/pull/14507/files#diff-ae8b99a6c960a65f7742f67c1270ddc6cc1c967c038562282c1ee31be86f938aR20). Doesn't seem useful to me to wrap it up again with more verbosity.